feat: Adding tickCount to TimerComponent (#3552)
flame-engine/flame: doc/flame/other/util.md

}
```

```{note}

> A `Timer` or `TimerComponent` can repeat indefinitely by providing a

Reviewer: ```suggestion
A `Timer` or `TimerComponent` can repeat indefinitely by providing the
```

> `repeat: true` property to it or it can repeat a certain number of

Reviewer: ```suggestion
`repeat: true` argument or it can be repeated a certain number of
```

> times by using the `tickCount` property alongside the `repeat`.

Reviewer: ```suggestion
times by using the `tickCount` argument together with `repeat: true`.
```
